caikit/core/model_management/multi_model_initializer.py

```diff
@@ -0,0 +1,146 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The MultiModelInitializer configures a set of other model initializers that will be used
+in sequence to try loading models.
+
+Configuration for MultiModelInitializer lives under the config as follows:
+
+model_management:
+    initializers:
+        <initializer name>:
+            type: MULTI
+            config:
+                # Sequence of other initializer names to use in priority order
+                initializer_priority:
+                    - other_initializer1
+                    - other_initializer2
+"""
+# Standard
+from typing import Optional
+
+# First Party
+import aconfig
+import alog
+
+# Local
+from ...config import get_config
+from ..exceptions import error_handler
+from ..modules import ModuleBase, ModuleConfig
+from .model_initializer_base import ModelInitializerBase
+
+# NOTE: Top-level import done so that global MODEL_MANAGER can be used at
```

Comment:
Thanks for this!

Reply:
Totally not copied from the [multi_model_finder](https://github.com/caikit/caikit/blob/7f3e08ff238d36ac466c1a3ddf5f98c598fc3a2d/caikit/core/model_management/multi_model_finder.py#L43)........